### DISCUSSION TITLE
Struggling to install on MacOS

### DISCUSSION THREAD

**Discussion by radiatewishbone** [oobabooga/text-generation-webui#5960] [Q&A]
MacBook Pro M1 Pro, Sonoma 14.4.1

I followed along with the manual install via Conda perfectly, but then when I get to the "python server.py" section, I get this in Terminal:

"(textgen) My-MacBook-Pro:text-generation-webui myusername$ python server.py
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /Users/myusername/text-generation-webui/text-generation-webui/server.py:6  │
│ in <module>                                                                  │
│                                                                              │
│     5                                                                        │
│ ❱   6 import accelerate  # This early import makes Intel GPUs happy          │
│     7                                                                        │
╰──────────────────────────────────────────────────────────────────────────────╯
ModuleNotFoundError: No module named 'accelerate'
(textgen) My-MacBook-Pro:text-generation-webui myusername$ "

I'm new to this world so I have no idea how to fix this. Any help?

**Comment by blackmambaza** [comment]
In the root folder of TGWUI, don't forget to run:  ./cmd_macos.sh and then activate your conda environment.

If that doesn't work then go to /installer_files/env/bin/ and type:  ./python -m pip install accelerate

This will fix your immediate issue.
